src/polyfills.ts

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/no-unassigned-import */
+import "es-iterator-helpers/auto";
```

Comment:
Hmm - does this have to be added? What is missing here?

Reply:
This polyfills the iterator helpers proposal; the tsconfig adds the typing but it doesn't bring the implementation. This adds the implementation, bringing the ability to use things like `filter`, `find, and `map` on iterables, which saved me time on implementing them myself as I was needing or will need them.

Reply:
I believe Node 22 implements these calls natively.